### PR TITLE
aodn wave site names - only 1st letter upcase

### PR DESCRIPTION
**Pull request by ana-berger**

### CODE REVIEW

**Comment by ana-berger** [comment]
this was totally a chatGPT 'doing'!

File: workspace/AODN_WAVE/process/DataHarvest/HarvestFile_WaveGeneric_RT_0.1.item [hunk 1]
```diff
@@ -198,10 +198,10 @@
       <varTables sizeState="INTERMEDIATE" name="Var">
         <mapperTableEntries name="lon_avg" expression="(row1.geospatial_lon_min+row1.geospatial_lon_max)/2 " type="id_Double"/>
         <mapperTableEntries name="lat_avg" expression="(row1.geospatial_lat_min +row1.geospatial_lat_max )/2" type="id_Double"/>
-        <mapperTableEntries name="sitename" expression="StringHandling.DOWNCASE(row1.site_name) " type="id_String"/>
+        <mapperTableEntries name="sitename" expression="row1.site_name.toLowerCase().substring(0, 1).toUpperCase() + row1.site_name.toLowerCase().substring(1) " type="id_String"/>
```
Comment (bpasquer) [inline workspace/AODN_WAVE/process/DataHarvest/HarvestFile_WaveGeneric_RT_0.1.item:201]: there's no need to convert to lowercase first. It's done in the second part of the line

**Review by bpasquer — CHANGES_REQUESTED** [review]

File: workspace/AODN_WAVE/process/DataHarvest/HarvestFile_WaveGeneric_RT_0.1.item (hunk 1 above)
Comment (ana-berger) [reply workspace/AODN_WAVE/process/DataHarvest/HarvestFile_WaveGeneric_RT_0.1.item:201]: hum ok, I will test, but that's what chatGPT told me to do

Comment (ana-berger) [reply workspace/AODN_WAVE/process/DataHarvest/HarvestFile_WaveGeneric_RT_0.1.item]: > there's no need to convert to lowercase first. It's done in the second part of the line

done!

**Review by bpasquer — APPROVED** [review]

**Pull request merged**